MAUI/MauiAppWithBroker/AppCreationScripts/Cleanup.ps1

```diff
@@ -0,0 +1,87 @@
+﻿
+[CmdletBinding()]
+param(    
```

Comment:
>param [](http://example.com/codeflow?start=0&length=5)

this doesnt need a sample.json ? (i suppose so, but confirming)

Reply:
It shouldn't. The application itself doesn't require any configuration there neither.